website/static/js/onboarder.js

```diff
@@ -440,6 +445,24 @@ function OBUploaderViewModel(params) {
         }
         if (!self.dropzone.getQueuedFiles().length) {
             self.changeMessage('Please select at least one file to upload.', 'text-danger');
+
+            // Delete node only if a new project is created
+            if(self.newProjectName() != null)
```

Comment:
Oh, I see, we delete the project if the user tries to start an upload to a new project without first selecting a file.

Is there any way we can prevent the creation of a new project before we get to this point?